big_tests/tests/graphql_SUITE.erl

```diff
@@ -0,0 +1,205 @@
+-module(graphql_SUITE).
```

Comment:
That should be added into `big_tests/default.spec`.
I think that is the reason the test coverage is so low